src/main/java/com/dumbdogdiner/stickyapi/bukkit/command/tabcomplete/NonCompletingTabExecutor.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.command.tabcomplete;
+
+
+import com.dumbdogdiner.stickyapi.common.arguments.Arguments;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+import java.util.List;
+
+public class NonCompletingTabExecutor implements TabExecutor {
```

Comment:
The fact that we need an entire file for what is effectively a lambda that returns an empty list amuses me